sites/cnfpt/src/backend/base/storage.py

```diff
@@ -58,4 +58,4 @@ class MediaStorage(S3Boto3Storage):
     bucket_name = getattr(settings, "AWS_MEDIA_BUCKET_NAME", None)
     custom_domain = getattr(settings, "CDN_DOMAIN", None)
     file_overwrite = False
-    location = settings.MEDIA_URL
```

Comment:
Why not just update `MEDIA_URL` in settings.py?

Reply:
I did not manage to do it with "location" as a class property because it was evaluated too early. 
I reverted to the AWS_LOCATION setting and it works. :+1: